File: 2021/05/11/main.py
class Solution:
    def maxScore(self, cardPoints: List[int], k: int) -> int:
        t = sum(cardPoints[:k])
        r = t
        for i in range(k):
            t = t - cardPoints[k-i-1] + cardPoints[-i-1]
            r = max(r, t)
        return r

File: 2021/05/11/main.go
package main

func maxScore(cardPoints []int, k int) int {
	var t = 0
	for i := 1; i <= k; i++ {
		t += cardPoints[i-1]
	}
	var r = t
	var l = len(cardPoints)
	for i := 1; i <= k; i++ {
		t = t - cardPoints[k-i] + cardPoints[l-i]
		if t > r {
			r = t
		}
	}
	return r
}
